File: 05 - Python/04 - Crescente.py
print("Digite dois numeros: ")
primeiro = int(input())
segundo = int(input())

while (primeiro != segundo):
    if (primeiro < segundo):
        print("CRESCENTE!")
    else:
        print("DECRESCENTE!")

    print("Digite outros dois numeros: ")
    primeiro = int(input())
    segundo = int(input())

File: 04 - Csharp/04 - Crescente.cs
using System;

namespace Programa
{
    class program
    {
        static void Main(string[] args)
        {
            int primeiro, segundo;

            Console.WriteLine("Digite dois numeros: ");
            primeiro = int.Parse(Console.ReadLine());
            segundo = int.Parse(Console.ReadLine());

            while (primeiro != segundo)
            {
                if ( primeiro < segundo)
                {
                    Console.WriteLine("CRESCENTE!");
                }
                else
                {
                    Console.WriteLine("DECRESCENTE!");
                }

                Console.WriteLine("Digite outros dois numeros: ");
                primeiro = int.Parse(Console.ReadLine());
                segundo = int.Parse(Console.ReadLine());
            }
        }
    }
}